### DISCUSSION TITLE
Dropdown show and site

### DISCUSSION THREAD

**Discussion by robpearmain** [n8design/htwoo#5] [Q&A]
This is awesome work, thanks so much for starting htwoo.

In your example you showed a drop down list, I guess in the example you are using a script to show and hide the drop down list on button click.

It is possible to style <select><option> etc, or do you have to construct it as <ul><li> and add JS to show and hide on dropdown click?

Are you planning on adding react examples for controls, or are you leaving it to the community to construct UI components.

Really excellent work

**Comment by StfBauer** [comment]
It is in the planning that we support ReactJS components too in future. These then will be built on HTML/CSS and JavaScript.

hTWOo core will be always the base then for those implementations.

There is still some documentation work too do.
